src/components/FreakTile.js

```diff
@@ -0,0 +1,20 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import './FreakTile.css';
+
+function FreakTile({ id, picture, name }) {
+  return (
+    <a href={ `https://localhost:4200/freaks/${ id }` } className="tile">
```

Comment:
Do not put links to localhost. instead use a relative path, like
`/freaks/${ id }`